website/docs/r/nsxv_firewall.markdown

```diff
@@ -0,0 +1,255 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_nsxv_firewall"
+sidebar_current: "docs-vcd-resource-nsxv-firewall"
+description: |-
+  Provides a vCloud Director firewall resource for advanced edge gateways (NSX-V). This can be used
+  to create, modify, and delete firewall rules.
+---
+
+# vcd\_nsxv\_firewall
```

Comment:
I would recommend in documentation refer as `Advanced edge gateway (NSX-V)`

Reply:
It's just my humble view point view:) @lvirbalas what is your view?

Reply:
Two things here:
1. Either GitHub misbehaves, but we have agreed that it should "vcd_nsxv_firewall_**rule**" and I even saw commits changing to that.
2. I agree that we should note that this is for advanced EG, but it is already here a line below, no?

Reply:
1. I think it misbehaves. If I open the PR - it is changed.
2. @vbauzysvmware if you can point it more explicitly. I think it is noted in https://github.com/terraform-providers/terraform-provider-vcd/pull/341/files#diff-215be64c11703733b065d8ed7b543c5fR12